language_detector/languages.py

```diff
@@ -3,7 +3,23 @@
 LANGUAGES = [
```

Comment:
It wasn't 100% necessary to keep the dict format here. It probably made more work for you guys. Imagine how much easier it would be if you just had a dict like this:
```python
{
  'Spanish':[LIST OF COMMON WORDS]
  'German':[LIST OF COMMON WORDS]
  'English':[LIST OF COMMON WORDS]
}
```